packages/ci/src/utils/test-run.ts

```diff
@@ -205,12 +214,12 @@ async function display(pipeline: Heroku.Pipeline, number: number, command: Comma
     let {body: testNodes} = await command.heroku.get<Heroku.TestNode[]>(`/test-runs/${testRun.id}/test-nodes`)
     let firstTestNode = testNodes[0]
 
-    if (firstTestNode) { await stream(firstTestNode.setup_stream_url!) }
+    if (firstTestNode) {await stream(firstTestNode.setup_stream_url!)}
 
-    if (testRun) { testRun = await waitForStates(RUNNING_STATES, testRun, command) }
-    if (firstTestNode) { await stream(firstTestNode.output_stream_url!) }
+    if (testRun) {testRun = await waitForStates(RUNNING_STATES, testRun, command)}
+    if (firstTestNode) {await stream(firstTestNode.output_stream_url!)}
 
-    if (testRun) { testRun = await waitForStates(TERMINAL_STATES, testRun, command) }
+    if (testRun) {testRun = await waitForStates(TERMINAL_STATES, testRun, command)}
 
     // At this point, we know that testRun has a finished status,
     // and we can check for exit_code from firstTestNode
```

Comment:
@talbright one more white space clean-up ;).  

Please add back the space after `{` and before `}`, for all the block statements.  For example, it should be:
```
if (firstTestNode) { await stream(firstTestNode.setup_stream_url!) }
```

Reply:
Did you guys update the linter? Originally this is how I had it but I ended up having to make changes to vscode settings to remove those spaces.